kornia/feature/loftr/loftr.py

```diff
@@ -74,7 +74,9 @@ class LoFTR(nn.Module):
         >>> out = loftr(input)
     """
 
-    def __init__(self, pretrained: Optional[str] = 'outdoor', config: Dict = default_cfg):
+    def __init__(self, pretrained: Optional[str] = 'outdoor', config: Dict = None):
+        if config is None:
+            config = default_cfg
```

Comment:
@ducha-aiki can we check in this PR what's the cause of  https://github.com/kornia/kornia/runs/3582943510?check_suite_focus=true and how we can fix ?

Reply:
No - the problem is the pytorch version 1.6.0 not supporting the pretrained weights as they are packed, as I told you before.

Reply:
ok, I'll send a quick fix now